api/handler/load.go

```diff
@@ -39,8 +41,16 @@ func LoadArcHandler(e *echo.Echo, logger utils.Logger) error {
 	}
 
 	var apiHandler api.HandlerInterface
+	defaultPolicySetting, found := gocore.Config().Get("defaultPolicy")
+	var defaultPolicy *bitcoin.Settings
+	if found && defaultPolicySetting != "" {
+		if err = json.Unmarshal([]byte(defaultPolicySetting), &defaultPolicy); err != nil {
```

Comment:
Are you passing pointer to the pointer here? Is it what the function expects? Have you tested that part ? 🤔

Reply:
Good point! Will resolve and test.